web/docs/project/_baseDirEnvNote.md

```diff
@@ -0,0 +1,6 @@
+:::caution Setting the correct env variable
+
+When you set the `baseDir` option, make sure that the `WASP_WEB_CLIENT_URL` env variable also includes the base directory.
+
+For example, if you are serving your app from `https://example.com/my-app`, the `WASP_WEB_CLIENT_URL` should be set to `https://example.com/my-app`.
```

Comment:
in first line: "When you set" -> "If you set"
in second line: "the base" -> "that base"
third line: I added a counter example.

```
If you set the `baseDir` option, make sure that the `WASP_WEB_CLIENT_URL` env variable also includes that base directory.

For example, if you are serving your app from `https://example.com/my-app`, the `WASP_WEB_CLIENT_URL` should be also set to `https://example.com/my-app`, and not just `https://example.com`.
```

Reply:
@Martinsos 
![Screenshot 2023-10-25 at 11 49 42](https://github.com/wasp-lang/wasp/assets/2223680/59193d1e-33aa-41ac-a48a-dd7276ad0bcf)

Reply:
I am aware of this feature :D, but it isn't always offered -> sometimes I can add a suggestions, sometimes I can't. This time I couldn't, so therefore I opted for this.

Reminder: don't forget to do our github review thing -> leave an emoji or comment so other person knows that you saw / acknowledge / solved the comment.